rlp/codec.py

```diff
@@ -15,20 +15,64 @@
 from rlp.utils import ALL_BYTES
 
 
-def decode_raw(item, strict, preserve_per_item_rlp):
-    try:
-        return rusty_rlp.decode_raw(item, strict, preserve_per_item_rlp)
-    except (TypeError, rusty_rlp.DecodingError) as e:
-        raise DecodingError(e, item)
-
-
-def encode_raw(obj):
-    try:
-        if isinstance(obj, bytearray):
-            obj = bytes(obj)
-        return rusty_rlp.encode_raw(obj)
-    except(rusty_rlp.EncodingError) as e:
-        raise EncodingError(e, obj)
+try:
+    import rusty_rlp
+except ImportError as e:
+    import logging
+    from rlp.atomic import (
+        Atomic,
+    )
+    logger = logging.getLogger()
+    logger.warning(
+        "Rust backend not supported. Falling back to slow RLP implementation. "
+        "If this is a 64-bit system, you may have a 32-bit Python installed."
+    )
```

Comment:
The warning can be observed by running `tox -e py38-core-fallback -- --log-cli-level WARNING`

I think it is especially useful to avoid situations where people are on a 64-bit system but have installed a 32-bit Python.